patches/react-native+0.48.4.patch

```diff
@@ -0,0 +1,136 @@
+diff --git a/node_modules/react-native/Libraries/Alert/AlertIOS.js b/node_modules/react-native/Libraries/Alert/AlertIOS.js
+index 4a81430..ddabfd0 100644
+--- a/node_modules/react-native/Libraries/Alert/AlertIOS.js
++++ b/node_modules/react-native/Libraries/Alert/AlertIOS.js
+@@ -207,11 +207,11 @@ class AlertIOS {
+         'keyboardType) and the old syntax will be removed in a future version.');
+ 
+       var callback = type;
+-      var defaultValue = message;
++      var deprecateDefaultValue = message;
+       RCTAlertManager.alertWithArgs({
+         title: title || undefined,
+         type: 'plain-text',
+-        defaultValue,
++        deprecateDefaultValue,
+       }, (id, value) => {
+         callback(value);
+       });
+@@ -257,4 +257,4 @@ class AlertIOS {
+   }
+ }
+ 
+-module.exports = AlertIOS;
++module.exports = AlertIOS;
+\ No newline at end of file
+diff --git a/node_modules/react-native/Libraries/Core/Timers/JSTimers.js b/node_modules/react-native/Libraries/Core/Timers/JSTimers.js
+index 73b6c12..436f604 100644
+--- a/node_modules/react-native/Libraries/Core/Timers/JSTimers.js
++++ b/node_modules/react-native/Libraries/Core/Timers/JSTimers.js
+@@ -241,7 +241,7 @@ const JSTimers = {
+   setTimeout: function(
+     func: Function,
+     duration: number,
+-    ...args?: any
++    ...args: any
+   ): number {
+     if (__DEV__ && IS_ANDROID && duration > MAX_TIMER_DURATION_MS) {
+       console.warn(
+@@ -267,7 +267,7 @@ const JSTimers = {
+   setInterval: function(
+     func: Function,
+     duration: number,
+-    ...args?: any
++    ...args: any
```

Comment:
This does not get stripped by Flow and the TS parser complains about it, saying it’s not according to spec, so probably isn't?

Reply:
TS interfaces can't do this feature, yeah

Reply:
What’s weird is that the tool to remove Flow is jut based on the Babel plugins and it leaves this token in. Maybe the Babel 7 version of the plugin regressed? 🤔